src/ios/CDVCamera.m

```diff
@@ -525,8 +525,24 @@ - (void)resultForImage:(CDVPictureOptions*)options info:(NSDictionary*)info comp
 
 - (CDVPluginResult*)resultForVideo:(NSDictionary*)info
 {
-    NSString* moviePath = [[info objectForKey:UIImagePickerControllerMediaURL] absoluteString];
-    return [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsString:moviePath];
+    NSURL *url = [info objectForKey:UIImagePickerControllerMediaURL];
+
+    NSData *urlData = [NSData dataWithContentsOfURL:url];
+    NSArray   *paths = NSSearchPathForDirectoriesInDomains(NSDocumentDirectory, NSUserDomainMask, YES);
+    NSString  *documentsDirectory = [paths objectAtIndex:0];
+    NSString  *newfilePath = [NSString stringWithFormat:@"%@/%@", documentsDirectory, [url lastPathComponent]];
+
+     BOOL fileExists = [[NSFileManager defaultManager] fileExistsAtPath:newfilePath];
+     if (fileExists){
```

Comment:
Is there a reason this isn't just:

```objective-c
if ([[NSFileManager defaultManager] fileExistsAtPath:newfilePath] == YES) {
```

Reply:
yes, it reads better

Reply:
Can there be a space after the closing parens? i.e. `if (fileExists){` to `if (fileExists) {`